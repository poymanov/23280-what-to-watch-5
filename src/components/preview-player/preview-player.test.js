import React from "react";
import renderer from "react-test-renderer";
import PreviewPlayer from "./preview-player";

const movie = {
  id: 1,
  name: `test`,
  posterImage: `test`,
  previewImage: `test`,
  backgroundImage: `test`,
  backgroundColor: `test`,
  videoLink: `test`,
  previewVideoLink: `test`,
  description: `test`,
  rating: 1,
  scoresCount: 1,
  director: `test`,
  starring: [`test`],
  runTime: 1,
  genre: `test`,
  released: 1,
  isFavorite: false,
};

it(`Should PreviewPlayer render correctly`, () => {
  const tree = renderer
    .create(<PreviewPlayer movie={movie}/>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});


