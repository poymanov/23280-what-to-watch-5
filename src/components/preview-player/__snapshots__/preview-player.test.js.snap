// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should PreviewPlayer render correctly 1`] = `
<video
  autoPlay={true}
  height="175"
  muted={true}
  poster="test"
  src="test"
  width="280"
/>
`;
